app/assets/stylesheets/govuk_publishing_components/components/_accordion.scss

```diff
@@ -1,25 +1,231 @@
 @import "govuk/components/accordion/accordion";
 
-.gem-c-accordion {
-  .govuk-accordion__section-heading {
-    // this is a temporary addition to fix the line height when it
-    // is being overridden by styles from govuk-template in collections
-    @include govuk-media-query($until: desktop) {
-      @include govuk-font(24, $weight: bold, $line-height: 1.6);
+$gem-c-accordion-border-width: 3px;
+$gem-c-accordion-bottom-border-width: 2px;
+
+// Buttons within the sections don’t need default styling
+.gem-c-accordion__section-button {
+  @include govuk-font($size: 24, $weight: bold);
+  display: inline-block;
+  margin-bottom: 0;
+  padding-top: govuk-spacing(3);
+}
+
+.gem-c-accordion__section-header {
+  padding-top: govuk-spacing(3);
+  padding-bottom: govuk-spacing(3);
+}
+
+.js-enabled {
+  .gem-c-accordion {
+
+    border-bottom: $gem-c-accordion-bottom-border-width solid $govuk-border-colour;
+  }
+
+  .gem-c-accordion__controls {
+    text-align: left;
+  }
+
+  .gem-c-accordion__open-all {
+
+    position: relative;
+    z-index: 1;
+    border-width: 0;
+    color: $govuk-link-colour;
+    background: none;
+    cursor: pointer;
+    -webkit-appearance: none;
+    margin: 0 0 govuk-spacing(4) 0;
+    padding: 0 0 govuk-spacing(1) 0;
+
+    @include govuk-font($size: 16);
+    @include govuk-link-common;
+    @include govuk-link-style-default;
+
+    // Remove default button focus outline in Firefox
+    &::-moz-focus-inner {
+      padding: 0;
+      border: 0;
     }
   }
-}
 
-.govuk-accordion--condensed {
-  .govuk-accordion__section-button {
-    @include govuk-media-query($from: tablet) {
-      @include govuk-font($size: 19, $weight: bold);
+  // Focus state to change chervon icon to black
+  .gem-c-accordion__open-all:focus {
+    .gem-c-accordion-nav__chevron--stroke {
+      stroke: govuk-colour("black");
+    }
+  }
+
+  .gem-c-accordion__section-heading {
+    // Override browser defaults to ensure consistent element height
+    @include govuk-font(24);
+
+    margin-top: 0; // Override browser default
+    margin-bottom: 0; // Override browser default
+  }
+
+  // Section headers have a pointer cursor as an additional affordance
+  .gem-c-accordion__section-header {
+    position: relative;
+    // Safe area on the right to avoid clashing with icon
+    padding-right: 40px;
+    border-top: $gem-c-accordion-bottom-border-width solid $govuk-border-colour;
+    color: $govuk-link-colour;
+    cursor: pointer;
+  }
+
+  // For devices that can't hover such as touch devices,
+  // remove hover state as it can be stuck in that state (iOS).
+  @media (hover: none) {
```

Comment:
Ooooooooo...

Reply:
Design System port not mine - but a f'sure, I had a similar "Oooo" reaction  😁